config/default/kustomization.yaml

```diff
@@ -22,7 +22,7 @@ resources:
 # [CERTMANAGER] To enable cert-manager, uncomment all sections with 'CERTMANAGER'. 'WEBHOOK' components are required.
 #- ../certmanager
 # [PROMETHEUS] To enable prometheus monitor, uncomment all sections with 'PROMETHEUS'.
-#- ../prometheus
+- ../prometheus
```

Comment:
This is the only thing I'm unsure about. This would enable the service monitor by default in kustomize releases as well. Not sure how it handles with missing CRDs in case the user doesn't have the prometheus operator set up

Reply:
Kustomize is actually defined in a different place, it's just that OLM is also using kustomize, but in a different structure.

I've reworked the e2e deployment to use the kustomize artifacts instead, which turns CI green again :)